web/themes/custom/recipe_theme/src/sass/view/_user_account_page.scss

```diff
@@ -0,0 +1,47 @@
+.user__name,
+.user__field-first-name {
+  .field__label,
+  .field__item {
+    text-align: center;
+    font-family: 'Dosis', sans-serif;
+    font-style: normal;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: $gray-800;
+  }
+  .field__label {
+    margin-top: 0.5rem;
+  }
+  .field__item {
+    background-color: $gray-200;
+    padding: 1rem auto;
+    border-radius: 0.4rem;
+    font-weight: 300;
+  }
+}
+
+.user__user-picture {
+  display: flex;
+  justify-content: center;
+}
+
+.user__field-introduction {
+  .field__label {
```

Comment:
This can be still further simplified. There are rules here which are the same as 

```sass
.user__name,
.user__field-first-name {
  .field__label {
    rules...
  }
}
```

Please combine the common rules.